app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -29,30 +40,59 @@ export const SearchPage: React.FC = () => {
   const [, store] = useAppStore();
 
   const api = useUsers();
+  const { query } = useParams();
+  const urlParams = new URLSearchParams(query);
+
+  const search = React.useMemo(
+    () =>
+      fromQueryString(query, {
+        arrays: ['sourceIds'],
+        numbers: ['sourceIds'],
+      }),
+    [query],
+  );
 
-  const urlParams = new URLSearchParams(window.location.search);
-  const queryText = urlParams.get('queryText');
+  const advancedSubFilter: IContentListFilter & Partial<IContentListAdvancedFilter> =
```

Comment:
where did the name `advancedSubFilter` come from?  What is it the `Sub` to?

Reply:
Was short for subscriber, I can see the confusion though - will change!

Reply:
I dont think it needs any reference to `susbcriber` in the name unless it's in the shared library and we already have an `advancedFilter` there?

Reply:
True, will change again :P